pages/docs/index.md

```diff
@@ -62,63 +124,257 @@ curl -X GET "https://<ENVIRONMENT_URL>/v1/user?id=PvRoVxxiJ2gul3qLwKOVaJ0HCNs1"
 			"registration_date": "2024-08-09T02:11:59.368403",
 			"in_lab_date": "2024-08-011T02:11:59.368403",
 			"processed_date": "2024-08-15T02:11:59.368403",
-			"barcode": "TU2Q6F72"
+			"barcode": "TU2Q6F72",
+			"health_status": {
+				"alcohol_consumption": "0",
+				"waking_up_condition": "Yes",
+				"date": "2024-11-27T00:00:00",
+				"medicine_consumption": "Yes",
+				"trouble_sleeping": "No",
+				"height_feet": 5.0,
+				"has_diabetes": "Yes",
+				"sleep_hours": "8",
+				"weight": 160.0,
+				"has_cancer": "No",
+				"physical_activity_days": "3",
+				"supplement_consumption": "Vitamin D",
+				"overall_health": "Good",
+				"id": 12,
+				"coffee_consumption": "Yes",
+				"mental_health": "Fair",
+				"user_id": null,
+				"kit_id": 14,
+				"tobacco_consumption": "No",
+				"social_life": "I'd rather not say",
+				"height_inches": 70.0,
+				"stress": "Not at all"
+			}
 		},
 		{
 			"sample_collection_date": "1098-10-10T00:00:00",
 			"registration_date": "1098-10-10T00:00:00",
 			"in_lab_date": "1098-10-15T00:00:00",
 			"processed_date": "1098-10-16T00:00:00",
-			"barcode": "TU33NQ34"
+			"barcode": "TU33NQ34",
+			"health_status": {
+				"alcohol_consumption": "0",
+				"waking_up_condition": "Yes",
+				"date": "2024-11-27T00:00:00",
+				"medicine_consumption": "Yes",
+				"trouble_sleeping": "No",
+				"height_feet": 5.0,
+				"has_diabetes": "Yes",
+				"sleep_hours": "8",
+				"weight": 160.0,
+				"has_cancer": "No",
+				"physical_activity_days": "3",
+				"supplement_consumption": "Vitamin D",
+				"overall_health": "Good",
+				"id": 12,
+				"coffee_consumption": "Yes",
+				"mental_health": "Fair",
+				"user_id": null,
+				"kit_id": 14,
+				"tobacco_consumption": "No",
+				"social_life": "I'd rather not say",
+				"height_inches": 70.0,
+				"stress": "Not at all"
+			}
 		}
 	]
 }
 ```
 
 &nbsp;
-### `POST /v1/user`
+### `POST /v1/kits` (Only staging yet)
 
-Creates a new TruMe user.
+Creates a new kit.
```

Comment:
Maybe this is a bit misleading. I'd say associates a kit with an user.  Also, /v1/kits can be a bit confusing too. Maybe /v1/kits/register?

Reply:
Saying this because if I read /v1/kits for post, I'd think I could create kits through the endpoint!